addons/disable-sprite-wobble/addon.json

```diff
@@ -48,8 +48,8 @@
       "default": "none"
     }
   ],
-  "tags": ["editor", "recommended"],
-  "enabledByDefault": false,
+  "tags": ["editor", "recommended", "minor"],
```

Comment:
This is the only addon with a setting and the new tag.

Reply:
TODO:
- [ ] If the `disable-sprite-wobble` addon gets enabled automatically, the `"mode"` setting should be reset.